cmake_modules/dependencies/el5.cmake

```diff
@@ -1 +1 @@
-SET_DEPENDENCIES ( CPACK_RPM_PACKAGE_REQUIRES boost141-regex openldap libicu m4 libtool libxslt libxml2 gcc-c++ openssh-server openssh-clients expect libarchive rsync apr apr-util zip tbb )
+SET_DEPENDENCIES ( CPACK_RPM_PACKAGE_REQUIRES boost141-regex openldap libicu m4 libtool libxslt libxml2 gcc-c++ openssh-server openssh-clients expect libarchive rsync apr apr-util zip tbb numactl-devel )
```

Comment:
Probably dont need -devel package for installation, just numactl.  But do need it for building so its probably fine this way.  But inconsistent with Debian below where its not the -dev package listed.